test/e2e/main.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"flag"
+	"github.com/golang/glog"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+
+const (
+	namespace = "openshift-cluster-api"
+)
+
+var (
+	F *Framework
+)
+
+type Framework struct {
+	Client client.Client
+}
+
+func newClient() error {
```

Comment:
Is there any reason this could not become just `init()`? Then you don't have to worry about ordering and `F` will be setup once only.